src/main/java/at/hannibal2/skyhanni/features/garden/FarmingFortuneDisplay.kt

```diff
@@ -172,28 +173,42 @@ object FarmingFortuneDisplay {
 
         list.add(
             Renderable.string(
-                "§6Farming Fortune§7: §e" + if (!recentlySwitchedTool && farmingFortune != -1.0) {
-                    farmingFortune.roundTo(0).addSeparators()
-                } else "§7" + (displayCrop.getLatestTrueFarmingFortune()?.addSeparators() ?: "?"),
+                (if (config.compactFormat) "§6FF§7: " else "§6Farming Fortune§7: ") + (if (ffReduction > 0) "§c" else "§e") +
```

Comment:
I would suggest giving descriptive names for values here instead of doing this all in blocks - all of this compacted like this is gonna make maintenance hellish.

Reply:
Can you give an example, I'm not sure I follow

Reply:
Take a look at the compact stuff we do in HoppityEggCompactChat - essentially assigning vals to be the outputs of a boolean condition, then combining those conditional strings at the end, instead of stringing together a bunch of statements.

```
val descriptiveVariable =  if (!recentlySwitchedTool && farmingFortune != -1.0) else "§7"
...
val final text = "§6Farming Fortune§7: §e$descriptiveVariable..."
```

Doing this outside of an IDE on a laptop screen is only going to go so well, but I hope this makes sense.